src/components/shared/Navbar.js

```diff
@@ -43,15 +43,22 @@ const URL = ({ link, children }) => {
 
 // titles: { left: [{ text, link }*], right: [{ text, link }*] }
 
-export const NavBar = ({ titles, font }) => {
+export const NavBar = ({ titles, font, bgColor, fontColor, img}) => {
   const { left, right } = titles
 
+  if (!bgColor) {
```

Comment:
this is not the right way of using optional parameters, check this out https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters. In other words, it sld be `bgColor='#FFFFFF'` on line 46